src/JaxkDev/Vehicles/Handlers/EventHandler.php

```diff
@@ -107,6 +136,12 @@ public function onEntityDamageEvent(EntityDamageByEntityEvent $event): void{
 							else {
 								$entity->close();
 								$attacker->sendMessage(Main::$prefix . "'" . $entity->getVehicleName() . "' has been removed.");
+                                if (($inv = $attacker->getInventory()) == null) {
+                                    $attacker->getLevel()->dropItem(Position::fromObject($attacker->add(0.5, 0.75, 0.5)), ($item));
+                                } else {
+                                    $attacker->getInventory()->addItem($item);
+                                }
+                                $item->setCount(1);
```

Comment:
Useless line, after adding item doesn't matter what its count is its not used again.